### PR TITLE
Change Postman collection buttons to a link

### PR DESCRIPTION
**Pull request by marksou**
Postman buttons no longer work properly after the addition of the Dashboard API. Changing to a link to the collections as a whole to replace the buttons.

### CODE REVIEW

**Comment by letzya** [comment]
This page https://deploy-preview-2023--tyk-docs.netlify.app/docs/nightly/tyk-apis/ looks a bit odd without the links to the collection, I think it needs some clarification, Maybe move the last section to the top and add some explanation in each section?

**Review by letzya — CHANGES_REQUESTED** [review]
This page https://deploy-preview-2023--tyk-docs.netlify.app/docs/nightly/tyk-apis/ looks a bit odd without the links to the collection, I think it needs some clarification, Maybe move the last section to the top and add some explanation in each section?

**Review by letzya — CHANGES_REQUESTED** [review]
Per my message

**Review by letzya — CHANGES_REQUESTED** [review]
Per my message

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ menu: none
 url: "/tyk-apis"
 ---
 
-Tyks own APIs allow you to access to the following
+Tyk's own APIs allow you to access to the following
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md]: Need more info here

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 2]
```diff
@@ -16,37 +16,13 @@ This API is very small, and has no granular permissions system. It is used purel
...
-<script type="text/javascript">
-  (function (p,o,s,t,m,a,n) {
-    !p[s] && (p[s] = function () { (p[t] || (p[t] = [])).push(arguments); });
-    !o.getElementById(s+t) && o.getElementsByTagName("head")[0].appendChild((
-      (n = o.createElement("script")),
-      (n.id = s+t), (n.async = 1), (n.src = m), n
-    ));
-  }(window, document, "_pm", "PostmanRunObject", "https://run.pstmn.io/button.js"));
-</script>
+Our Gateway API is available as a [Postman Collection](#tyk-postman-collections) that you can use to test our Gateway endpoints.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md]: Not just to test but also connect and control the gw.

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 3]
```diff
@@ -16,37 +16,13 @@ This API is very small, and has no granular permissions system. It is used purel
...
 
 ## [Tyk Dashboard API](/docs/tyk-dashboard-api/)
 
 The Tyk Dashboard API allows much more fine-grained, secure and multi-user access to your Tyk cluster, and should be used to manage a database-backed Tyk node. The Tyk Dashboard API works seamlessly with the Tyk Dashboard (the two come bundled together).
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md]: Can we please remove "the two of them ..."

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 4]
```diff
@@ -55,3 +31,10 @@ The Dashboard Admin API is a special bootstrapping API that can be used to set u
 ## [Tyk Portal API](/docs/tyk-portal-api/)
 
 The Tyk Portal API covers all available endpoints for your developer portal.
+
+## Tyk Postman Collections
+
+We have a Postman public workspace where you can view, fork and test our Gateway and Dashboard APIs.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md]: This sentence is being repeated 3-4 times...

**Review by letzya — CHANGES_REQUESTED** [review]
Per my comments

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 5]
```diff
@@ -5,7 +5,14 @@ menu: none
 url: "/tyk-apis"
 ---
 
-Tyks own APIs allow you to access to the following
+We have our own APIs and a Postman public workspace.
+
+## Tyk Postman Collections
+
+We have a Postman public workspace where you can view, fork and test our Gateway and Dashboard APIs.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:12]: `We` again. Needs rewriting

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 6]
```diff
@@ -5,7 +5,14 @@ menu: none
 url: "/tyk-apis"
 ---
 
-Tyks own APIs allow you to access to the following
+We have our own APIs and a Postman public workspace.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:8]: This page is not about postman it’s about our APIs. postman is just a tool to view them. Also needs to be 2-3 sentences

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 7]
```diff
@@ -16,42 +23,18 @@ This API is very small, and has no granular permissions system. It is used purel
...
-<script type="text/javascript">
-  (function (p,o,s,t,m,a,n) {
-    !p[s] && (p[s] = function () { (p[t] || (p[t] = [])).push(arguments); });
-    !o.getElementById(s+t) && o.getElementsByTagName("head")[0].appendChild((
-      (n = o.createElement("script")),
-      (n.id = s+t), (n.async = 1), (n.src = m), n
-    ));
-  }(window, document, "_pm", "PostmanRunObject", "https://run.pstmn.io/button.js"));
-</script>
+The Tyk Dashboard API allows much more fine-grained, secure and multi-user access to your Tyk cluster, and should be used to manage a database-backed Tyk node. The Tyk Dashboard API works seamlessly with the Tyk Dashboard.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:30]: `The Tyk Dashboard API allows much more fine-grained` - full RBAC, not just "more"

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 8]
```diff
@@ -16,42 +23,18 @@ This API is very small, and has no granular permissions system. It is used purel
...
 
 ## [Tyk Dashboard Admin API](/docs/dashboard-admin-api/)
 
-The Dashboard Admin API is a special bootstrapping API that can be used to set up and provision a Tyk Dashboard instance without the command line and is used by the bootstrap scripts that come with a Tyk On-Premises installation.
+The Dashboard Admin API is a special bootstrapping API that can be used to set up and provision a Tyk Dashboard instance without the command line and is used by the bootstrap scripts that come with a Tyk On-Premises installation. There is no public Postman collection for this API.
 
 ## [Tyk Portal API](/docs/tyk-portal-api/)
 
 The Tyk Portal API covers all available endpoints for your developer portal.
+
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:40]: Can we remove this section, we don't have this as a separate api or collection.

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 9]
```diff
@@ -16,42 +23,18 @@ This API is very small, and has no granular permissions system. It is used purel
...
-<script type="text/javascript">
-  (function (p,o,s,t,m,a,n) {
-    !p[s] && (p[s] = function () { (p[t] || (p[t] = [])).push(arguments); });
-    !o.getElementById(s+t) && o.getElementsByTagName("head")[0].appendChild((
-      (n = o.createElement("script")),
-      (n.id = s+t), (n.async = 1), (n.src = m), n
-    ));
-  }(window, document, "_pm", "PostmanRunObject", "https://run.pstmn.io/button.js"));
-</script>
+This allows you to test, connect to and control your Gateway.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:26]: Add a link to the api page https://deploy-preview-2023--tyk-docs.netlify.app/docs/nightly/tyk-gateway-api/

explain that this is the latest released version of our API and if you are a postman user, click on the button above to get to the official tyk gateway collection.
(same for the dashboard)

File: tyk-docs/content/tyk-apis/tyk-apis.md [hunk 10]
```diff
@@ -5,7 +5,14 @@ menu: none
 url: "/tyk-apis"
 ---
 
-Tyks own APIs allow you to access to the following
+We have our own APIs and a Postman public workspace.
+
+## Tyk Postman Collections
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-apis.md:10]: I think this section could look better as part of the one above, so maybe try to remove this title.

**Review by letzya — CHANGES_REQUESTED** [review]
Per my comments

**Review by letzya — CHANGES_REQUESTED** [review]
.

File: tyk-docs/content/tyk-apis/tyk-apis.md (hunk 9 above)
Comment (marksou) [reply tyk-docs/content/tyk-apis/tyk-apis.md:26]: The title for each section is a link to the appropriate API

**Review by letzya — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**